src/task_change.rs

```diff
@@ -413,97 +473,119 @@ pub fn compute_changeset(
     let mut from = from.into_iter().map(Some).collect::<Vec<Option<Task>>>();
 
     // Extract changed, new, and deleted tasks
-    let matches = matching
+    let mut matches = matching
         .into_iter()
         .enumerate()
         .filter_map(|(i, x)| x.map(|x| (i, x)))
-        .map(|(i, j)| (from[i].take().unwrap(), to[j].take().unwrap()))
-        .collect::<Vec<_>>();
-
-    let new_tasks = to.into_iter().flat_map(|x| x);
+        .map(|(i, j)| {
+            let from = from[i].take().unwrap();
+            let to = to[j].take().unwrap();
+            let delta = if from.tags.get("rec").is_some() {
+                Recurred(vec![to])
```

Comment:
The filtering is indeed done later. I don't want to rename it because once it has been outputted to the user, it is indeed correctly capturing tasks that have been recurred.

Reply:
What I mean is that currently this test passes:
```yaml
recurrable_changed:
  allowed_divergence: 40

  from:
    - hello rec:+1d

  to:
    - hella rec:+1d

  new: []

  changes:
    - Recurred:
      -
        - Subject("hello", "hella")
```

Which in my current understanding of the code, I take to mean that `Recurred` actually means “recurrable”?

As for the output to the user, I can't find the place where there's another layer of filtering… maybe it's in one of the follow-up PRs I haven't yet looked at? All I'm finding is this where the two are treated almost the same, and from my understanding from there on they are used only as iterators?
```rust
Changed(t) => Changed(changes_between(&x.orig, &t, true)),
Recurred(changes) => {
    // TODO: compute changes more cleverly
    let changes = changes
        .iter()
        .enumerate()
        .map(|(i, to)| changes_between(&x.orig, to, i == 0))
        .collect::<Vec<_>>();
    Recurred(changes)
}
```

Reply:
Ah, you're right, it seem that I only fix this in the next PR

Reply:
I changed that

Reply:
Great, thanks!